docs/troubleshoot/ze/ze-known-limits.md

```diff
@@ -0,0 +1,26 @@
+# Known Zowe Explorer limits
+
+### Mismatched credentials when using Zowe Explorer and Zowe CLI
+
+#### Limitation
+
+Mismatching credentials can potentially lock you out from using Zowe CLI and Zowe Explorer in Visual Studio Code with either Windows Subsystem for Linux (WSL) or Remote Secure Shell (SSH).
+
+WSL allows developers to run a Linux environment on Windows without the need for a separate virtual machine or dual booting. When using Zowe Explorer with WSL, two parallel processes take place: VS Code runs on the Windows operating system, while code execution and tooling happen within the Linux environment. 
+
+With Remote SSH, the network protocol provides users with a secure connection to a remote computer. When using this protocol with Zowe Explorer, you can securely connect to a remote machine or a remote development environment. 
+
+Both WSL and Remote SSH provide more tools for mainframe developers, but they also have limitations when it comes to credentials.
+
+Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, API Mediation Layer tokens &mdash; resides on the developer’s local machine, not the Linux environment or the remote server or virtual machine. This is because credential storage is managed by VS Code, which runs on Windows.
```

Comment:
Comment regarding "This is because credential storage is managed by VS Code, which runs on Windows."

VS Code can be run under Linux and MacOS as well. I think the key point is that VS Code will load credentials from the local filesystem (or whatever filesystem contains the OS that hosts VS Code) rather than from a WSL VM or remote SSH connection that VS Code could be using. Maybe @zFernand0 can check me on how this should be worded.

Reply:
You do have a good point.
I believe I suggested Windows thinking only about WSL(2).
But it's possible for someone on any flavor of Linux or MacOS to be running VSCode and connecting over remote SSH to another machine.

How about...
"This is because credential storage is managed by VS Code, which stores them using the host's Operating System Credential Manager"?

(or some other flavor) 😋